src/components/ToSTIX/Mapping.jsx

```diff
@@ -0,0 +1,356 @@
+                                </div>
+                                {mapping[objectName][fieldId].mapped_to.map(
+                                  (stixField) => {
+                                    const stixFieldTransformer = mapping[
+                                      objectName
+                                    ][fieldId].mapped_to
+                                      .filter(
+                                        (o) =>
+                                          o.id === stixField.id && o.transformer
+                                      )
+                                      .map((i) => i.transformer)[0];
+                                    let stixFieldReferences = mapping[
+                                      objectName
+                                    ][fieldId].mapped_to
+                                      .filter(
+                                        (o) =>
+                                          o.id === stixField.id && o.references
+                                      )
+                                      .map((i) => i.references);
+                                    stixFieldReferences =
+                                      stixField.key.endsWith("_refs")
+                                        ? stixFieldReferences[0]
+                                        : stixFieldReferences;
+                                    return (
+                                      <div key={fieldId + stixField.id}>
+                                        <div
+                                          className={`bx--row ${styles.object_item__field}`}
+                                        >
+                                          <div className={"bx--col-sm-1"}>
+                                            <TextInput
+                                              id={stixField.id + stixField.key}
+                                              labelText={""}
+                                              onChange={(e) => {
+                                                dispatch(
+                                                  updateMappingField(
+                                                    objectName,
+                                                    fieldId,
+                                                    stixField.id,
+                                                    e.target.value,
+                                                    "key"
+                                                  )
+                                                );
+                                              }}
+                                              size={"sm"}
+                                              value={stixField.key}
+                                            />
+                                          </div>
+                                          <div className={"bx--col-sm-1"}>
+                                            <Dropdown
```

Comment:
is it possible to add the option to select `none`? because if you choose one of the transformers you are unable to unselect it